CONTRIBUTING.md

```diff
@@ -1,495 +1 @@
-
-If you would like to apply to join our Issue Moderator team - which is a Core Team position - message [@BerkeleyTrue](https://gitter.im/berkeleytrue) with links to 5 of your pull requests that have been accepted and 5 issues where you have helped someone else through commenting or QA'ing.
-
-### How We Close Stale Issues
-
-We will close any issues or pull requests that have been inactive for more than 15 days, except those that match the following criteria:
-- bugs that are confirmed
-- pull requests that are waiting on other pull requests to be merged
-- features that are a part of a GitHub project
-
-### Next Steps
-
-#### If your PR is accepted
-
-Once your PR is accepted, you may delete the branch you created to submit it.
-This keeps your working fork clean.
-
-You can do this with a press of a button on the GitHub PR interface. You can
-delete the local copy of the branch with: `git branch -D branch/to-delete-name`
-
-#### If your PR is rejected
-
-Don't despair! You should receive solid feedback from the Issue Moderators as to
-why it was rejected and what changes are needed.
-
-Many Pull Requests, especially first Pull Requests, require correction or
-updating. If you have used the GitHub interface to create your PR, you will need
-to close your PR, create a new branch, and re-submit.
-
-If you have a local copy of the repo, you can make the requested changes and
-amend your commit with: `git commit --amend` This will update your existing
-commit. When you push it to your fork you will need to do a force push to
-overwrite your old commit: `git push --force`
-
-Be sure to post in the PR conversation that you have made the requested changes.
-
-### Other Resources
-
--   [Style Guide for freeCodeCamp
-    Challenges](https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/seed/challenge-style-guide.md)
-
--   [Searching for Your Issue on
-    GitHub](http://forum.freecodecamp.org/t/searching-for-existing-issues/19139)
-
--   [Creating a New GitHub
-    Issue](http://forum.freecodecamp.org/t/creating-a-new-github-issue/18392)
-
--   [Select Issues for Contributing Using
-    Labels](http://forum.freecodecamp.org/t/free-code-camp-issue-labels/19556)
-
--   [How to clone the freeCodeCamp website on a Windows
-    pc](http://forum.freecodecamp.org/t/how-to-clone-and-setup-the-free-code-camp-website-on-a-windows-pc/19366)
-
--   [How to log in to your local freeCodeCamp site - using
-    GitHub](http://forum.freecodecamp.org/t/how-to-log-in-to-your-local-instance-of-free-code-camp/19552)
-
--   [Writing great git commit
-    messages](http://forum.freecodecamp.org/t/writing-good-git-commit-messages/13210)
-
--   [Contributor Chat Support - For the freeCodeCamp repositories, and running a local
-    instance](https://gitter.im/FreeCodeCamp/Contributors)
+> Note: This guide is now [moved to the docs directory](/docs/).
```

Comment:
I think these are some really good improvements 👍 My only concern is that GitHub automatically refers to the CONTRIBUTING.md when creating a new PR or issue. If we link them to the folder, they might not now which file they're looking for.

Reply:
Yes, you are correct. That is the prime reason I am keeping this file and linking it to docs, I think there is worth in keeping all documentation neatly organised in docs folder.

Sure, many contributors will land on this file, from GitHub interface in create new issue page mostly,
but we can link to the correct files, from other places, for example the contribute.freecodecamp.org and the readme for example, which gets the most traffic.